corehq/apps/app_manager/models.py

```diff
@@ -5311,6 +5369,13 @@ def fetch_file(self, location):
 
         return location, content
 
+    def get_build_langs(self):
+        if self.build_profiles:
+            # return first profile, generated as part of lazy migration
+            return self.build_profiles[self.build_profiles.keys()[0]].langs
```

Comment:
This code makes an assumption that there is only one build profile. Raising an exception makes that assumption explicit and makes the code easier to analyze. Having a check in the view also makes sense because you can give a more informed and user friendly error message there.